### PR TITLE
feat(step_2): add bootstrap and language translation tools

### PR DESCRIPTION
**Pull request by AlexMomot-717**

### CODE REVIEW

File: locale/ru/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (svntmr) [inline locale/ru/LC_MESSAGES/django.po]: wdyt, does it make sense to replace the placeholders with actual values?

File: task_manager/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+{% load i18n %}
+<html lang="">
```
Comment (svntmr) [inline task_manager/templates/base.html]: let's add value for lang or remove it

File: task_manager/templates/base.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  <head>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>{% trans "Task manager Hexlet" %}</title>
+    {% load django_bootstrap5 %} {% bootstrap_css %} {% bootstrap_javascript %}
```
Comment (svntmr) [inline task_manager/templates/base.html:10]: [optional] wdyt, is it the right time to make `prettier` jinja-aware? [this](https://github.com/davidodenwald/prettier-plugin-jinja-template) might help

File: task_manager/templates/index.html [hunk 4]
```diff
@@ -1,80 +1,16 @@
...
-    ></script>
-  </head>
-  <body class="d-flex flex-column min-vh-100">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light ms-3">
-      <a class="navbar-brand" href="/">Менеджер задач</a>
-      <button
-        class="navbar-toggler"
-        data-bs-toggle="collapse"
-        data-bs-target="#navbarToggleExternalContent"
+{% extends "base.html" %} {% block content %} {% load i18n %}
```
Comment (svntmr) [inline task_manager/templates/index.html]: is it necessary to load i18n two times? you load it in `base.html`

File: locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
+"n%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || "
+"(n%100>=11 && n%100<=14)? 2 : 3);\n"
+
+#: task_manager/templates/base.html:9
+msgid "Task manager Hexlet"
```
Comment (svntmr) [inline locale/ru/LC_MESSAGES/django.po]: wdyt about `Hexlet task manager` instead?

File: task_manager/templates/index.html (hunk 4 above)
Comment (svntmr) [reply task_manager/templates/index.html]: yes, it is - see the [docs](https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#internationalization-in-template-code)

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -39,11 +39,13 @@ repos:
       - id: isort
         name: isort (python)
         args: ["--profile", "black", "--filter-files"]
-  - repo: local
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: "v1.10.0"
```
Comment (svntmr) [inline .pre-commit-config.yaml]: `mypy` is our local hook, it's repo shouldn't be changed - as far as I remember

File: .pre-commit-config.yaml [hunk 7]
```diff
@@ -64,3 +66,17 @@ repos:
     rev: "v4.0.0-alpha.8"
     hooks:
       - id: prettier
+        name: prettier
+        entry: npx prettier task_manager --write **/*.html --ignore-unknown
+        language: node
+        additional_dependencies: ["prettier@4.0.0-alpha.8"]
```
Comment (svntmr) [inline .pre-commit-config.yaml]: possible way to go is to make it a system hook and install node, prettier, and plugin on your own 
you're doing almost the same stuff now as plugin is installed locally

File: pyproject.toml [hunk 8]
```diff
@@ -23,7 +24,8 @@ gitlint = "^0.19.1"
 isort = "^5.13.2"
 mypy = "^1.10.0"
 flake8 = "^7.0.0"
-django-stubs = "^5.0.0"
+#django-stubs = "^5.0.2"
```
Comment (svntmr) [inline pyproject.toml]: could you please remove this line?

File: pyproject.toml [hunk 9]
```diff
@@ -37,6 +39,12 @@ line_length = 79
 python_version = 3.11
 strict = true
 
+[tool.django-stubs]
+django_settings_module = "task_manager.settings"
+
+[tool.poetry.dev-dependencies]
+django-stubs = {extras = ["compatible-mypy"], version = "^5.0.2"}
+
```
Comment (svntmr) [inline pyproject.toml]: should be removed as you have the same stubs listed in `[tool.poetry.group.dev.dependencies]` section

File: .pre-commit-config.yaml (hunk 7 above)
Comment (svntmr) [reply .pre-commit-config.yaml]: or smth like [this](https://adamj.eu/tech/2023/02/09/pre-commit-hooks-unsupported-tools/#node-hook-stylelint-a-css-linter) (I like it more)

File: .pre-commit-config.yaml [hunk 10]
```diff
@@ -60,7 +60,13 @@ repos:
...
     hooks:
-      - id: prettier
+      - id: htmlformatter
+        name: html prettier formatter
+        language: node
+        additional_dependencies:
+          - prettier@3.3.2
+          - prettier-plugin-jinja-template@1.4.0
+        entry: npx prettier task_manager --write **/*.html
+        types_or: [html]
```
Comment (svntmr) [inline .pre-commit-config.yaml]: let's move all `local` repo hooks into one place - we have `mypy` hook defined in the `local` repo as well

File: .prettierrc.js [hunk 11]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  useTabs: false,
+  plugins: [require("prettier-plugin-jinja-template")],
+  overrides: [
+    {
+      files: ["**/{templates,jinja2}/**/*.html"],
```
Comment (svntmr) [inline .prettierrc.js]: shouldn't it be just? 
```suggestion
      files: ["*.html"],
```
that's what jinja-template plugin recommends and I'm not sure if we're going to have any pure HTML files

**Review by svntmr — APPROVED** [review]
feel free to address the comments and merge

File: .prettierrc.js (hunk 11 above)
Comment (AlexMomot-717) [reply .prettierrc.js]: agree

**Pull request merged**